src/debugger/appWorker.ts

```diff
@@ -152,14 +155,14 @@ postMessage({workerLoaded:true});`;
         this.packagerLocalRoot = attachRequestArguments.localRoot;
         this.sourcesStoragePath = sourcesStoragePath;
         this.projectRootPath = projectRootPath;
-        console.assert(!!this.sourcesStoragePath, "The sourcesStoragePath argument was null or empty");
+        console.assert(!!this.sourcesStoragePath, localize("SourcesStoragePathIsNullOrEmpty", "The sourcesStoragePath argument was null or empty"));
```

Comment:
Are you sure we need to localize this? Could you please reproduce the situation when assert is occured and see what will you get in VS Code logs?

Reply:
That's how it would look after this change 
![image](https://user-images.githubusercontent.com/32372875/50152003-7d985100-02d3-11e9-91c6-f15180cfb859.png)

Reply:
All this entries is processes [by this function ](https://github.com/Microsoft/vscode-react-native/pull/853/files#diff-933d1a6506e93a563f1ed6dc80ff0f47L227)

Reply:
@SounD120 alright then